tests/foreman/ui/test_errata.py

```diff
@@ -133,7 +134,7 @@ def module_repos_col(module_org, module_lce):
 @pytest.fixture
 def vm(module_repos_col):
     """Virtual machine client using module_repos_col for subscription"""
-    with VirtualMachine(distro=module_repos_col.distro) as client:
+    with VMBroker(nick=module_repos_col.distro, host_classes={'host': ContentHost}) as client:
```

Comment:
I love this pattern! Do we have nicks defined in CI for all of these included in your PR?

Reply:
This is a great improvement to replace `Virtualmachine`.  Glad to see it's in use.